src/pranitaPJun20/MethodCalling.java

```diff
@@ -0,0 +1,81 @@
+/*Assignment 9 : 17th Jun 2020
+
+Design to test static, non static, return type concept.
+
+1) Method sum(), accept 3 parameters of type double & return answer
+2) Method average(), accept 3 parameters of type double & return answer
+3) Method isAnswerAboveExpecation(), take one parameter (answer which you received from average method) and return true if value is more then 50 else false.
+4) Method isEligible(),take one parameter (answer which you received from average method) return true if value is more then 50 or divisible by 2. Eligibility will be printed in main method. 
+5) Method getYourGrade(),take one parameter (answer which you received from average method) return "A Grade" if average is more then 80 else "B grade", main method will print grade received from getYourGrade() method.
+
+Rules : a) From main method, you will call average() and average() method will call sum() method.
+               Main method will not directly call sum method. 
+	    b) average() & sum() should be non static method.
+	    c) isAnswerAboveExpecation(), isEligible(), getYourGrade() method should be static method and      called from main method.
+            d) Class should not have any instance variable.  
+			
+*/
+package pranitaPJun20;
+
+public class MethodCalling {
+//Method sum(), accept 3 parameters of type double & return answer
+	double sum(double num1,double num2,double num3){
+		double sumLocal = num1+num2+num3;
+		//System.out.println("The sum is:"+sumLocal);
+		return sumLocal;	
+	}
+//Method average(), accept 3 parameters of type double & return answer
+	double average(double num1,double num2,double num3){
+		sum(80.5,90.9,78.8);
+		double averageLocal = (num1+num2+num3)/3;
+		//System.out.println("The average is:"+averageLocal);
```

Comment:
line number 28, you should capture it in a variable and line number 30 should be.
double totalSum = sum(80.5,90.9,78.8);
double averageLocal = totalSum/3

Reply:
Done